### PR TITLE
fix: Paz app UI updated

### PR DESCRIPTION
**Pull request by HaseebAsif**
What I have done in this PR:
- Made the design according to the prototype.
- Made the design responsive
- Added the button functionality (but its not working properly, you have to refresh the page to see the connected screen).

### CODE REVIEW

**Comment by HaseebAsif** [comment]
Please check now!

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
+import { useQuery } from '@apollo/react-hooks'
```
Comment (bobachef) [inline src/App.js:1]: remove this

File: src/App.js [hunk 2]
```diff
@@ -9,16 +8,19 @@ import ConnectedLockYourPazz from './components/LockPaz/index'
 import Footer from './components/Footer/index'
 import { useWeb3Context } from './contexts/ConnectWeb3'
 
-const App = () => {
+function App() {
```
Comment (bobachef) [inline src/App.js:11]: how come you decided to randomly change the way this was written?

File: src/components/PazDropDown/index.js [hunk 3]
```diff
@@ -3,18 +3,70 @@ import Dropdown from 'react-dropdown'
...
     { value: 'PAZ', label: 'PAZ', className: 'paz-dropdown-icon' },
     { value: 'BNB', label: 'BNB', className: 'bnb-dropdown-icon' },
     { value: 'BUSD', label: 'BUSD', className: 'busd-dropdown-icon' },
-
   ]
   const defaultOption = options[0]
   return (
     <div>
-      <Dropdown options={options} value={defaultOption} placeholder="Select an option" className="paz-dropdown" />
+      <Dropdown options={options} value={defaultOption} placeholder="Select an option" style={pazzdropdownClasses} />
```
Comment (bobachef) [inline src/components/PazDropDown/index.js:69]: how come your not using styled components here?

**Pull request closed without merging**